extension/js/common/core/attachment.ts

```diff
@@ -177,4 +177,62 @@ export class Attachment {
       return 'plainFile';
     }
   };
+
+  public isExecutableFile = () => {
+    return [
+      '.ade',
```

Comment:
It'd probably make sense to not repeat the leading dot (.) for every item in this list, that is, simply have a list like `'ade', 'adp', ...`

Reply:
I have simplified this one as suggested. thanks!